packages/core/admin/ee/admin/content-manager/pages/EditView/InformationBox/components/AssigneeSelect/AssigneeSelect.js

```diff
@@ -22,18 +24,30 @@ export function AssigneeSelect() {
     isSingleType,
     onChange,
   } = useCMEditViewDataManager();
+  const permissions = useSelector(selectAdminPermissions);
   const { formatMessage } = useIntl();
   const { formatAPIError } = useAPIErrorHandler();
   const toggleNotification = useNotification();
   const { put } = useFetchClient();
-  const { users, isLoading, isError } = useAdminUsers();
+  const {
+    allowedActions: { canReadUsers },
+  } = useRBAC({
+    ...permissions.settings.auditLogs,
```

Comment:
Do you mean `...permissions.settings.review-workflows` ? As I think the select should also be disabled without the review workflow settings permissions?

```suggestion
		...permissions.settings['review-workflows'],

```